compiler/rustc_codegen_cranelift/src/lib.rs

```diff
@@ -1,13 +1,16 @@
+// tidy-alphabetical-start
+#![allow(rustc::diagnostic_outside_of_impl)]
+#![allow(rustc::untranslatable_diagnostic)]
 #![cfg_attr(doc, allow(internal_features))]
-#![cfg_attr(doc, feature(rustdoc_internals))]
 #![cfg_attr(doc, doc(rust_logo))]
+#![cfg_attr(doc, feature(rustdoc_internals))]
+// Note: please avoid adding other feature gates where possible
```

Comment:
This comment has been duplicated.

Reply:
It was deliberate, one above and one below to increase chance of it being seen... I can remove the second one if you like, though.

Reply:
(I also considered adding a `<<<` marker to the first one and a `>>>` marker to the second, to make the intent clearer, though I didn't actually do it.)